manual/core/native_protocol/README.md

```diff
@@ -28,14 +28,21 @@ first node the driver connects to:
 |---------------------|-------------------------------------------------|
 | 2.1.x (DSE 4.7/4.8) | v3                                              |
 | 2.2.x               | v4                                              |
-| 3.x (DSE 5.0/5.1)   | v4                                              |
-| 4.x ²               | v5                                              |
+| 3.x (DSE 5.0/5.1)   | v4 (DSE_V1)³ ⁵                                  |
+| 4.x (DSE 6.0/6.8)²  | v5 (DSE_V2)⁴ ⁵                                  |
 
 *(1) for previous driver versions, see the [3.x documentation][driver3]*
```

Comment:
`[driver3]` is currently taking us to the following,

> _https://docs.datastax.com/en/developer/java-driver/3.5/manual/native_protocol/_

can we instead go to the following?

> _https://docs.datastax.com/en/developer/java-driver/3.10/manual/native_protocol/_

Reply:
Thanks for that catch. I'll update the link to point to the latest.